README.md

```diff
@@ -21,7 +21,8 @@ How to use the microservice
 ## Quickstart - Running the service with Docker
 
 1. Install Docker Compose: https://docs.docker.com/compose/install
-1. Create your [Azure Cosmos DB account](https://ms.portal.azure.com/#create/Microsoft.DocumentDB) with API type **DocumentDB**
+1. Create your [Azure Cosmos DB account](https://ms.portal.azure.com/#create/Microsoft.DocumentDB) with API type **SQL**
+1. Change the default consistency in your Azure Cosmos DB account to **STRONG**
```

Comment:
I don't think this is required, if it is we should change the code to take care of this details where needed. For instance we consume the storage with different patterns, where consistency expectations are different. The code should be resilient to this config and do the right thing.

Reply:
Unless CosmosDB is setup with strong consistency, the service will encounter this error right now: 
{
    "Message": "An error has occurred.",
    "ExceptionMessage": "ConsistencyLevel Strong specified in the request is invalid when service is configured with consistency level Session. Ensure the request consistency level is not stronger than the service consistency level.",
...
}

Reply:
I propose merging this PR to fix the docs so things aren't broken in master, and then considering the code change to make it more resilient separately.